share/cpt/berlin.cpt

```diff
@@ -140,7 +140,8 @@
 -0.027450	21/14/14	-0.019608	22/14/13
 -0.019608	22/14/13	-0.011764	23/13/11
 -0.011764	23/13/11	-0.003922	24/12/10
--0.003922	24/12/10	0.003922	25/12/9
+-0.003922	24/12/10	0		25/12/9
```

Comment:
The change still doesn't make sense to me.

The original z-slice is `-0.003922	24/12/10	0.003922	25/12/9`. The color of z=0 should be an interpolated color of `24/12/10` and `25/12/9`, right?

Reply:
Rounded to nearest integer there is no difference.